### PR TITLE
Config HTTP endpoints

### PR DESCRIPTION
**Pull request by rkapka**
**What type of PR is this?**

Other

**What does this PR do? Why is it needed?**

HTTP version of GET and POST for:
- "/eth/v1/config/spec"
- "/eth/v1/config/fork_schedule"

I also moved "/eth/v1/config/deposit_contract" to the `config` package.

**Which issues(s) does this PR fix?**

Part of #12970

### CODE REVIEW

File: beacon-chain/rpc/eth/config/handlers_test.go [hunk 1]
```diff
@@ -353,7 +383,9 @@ func TestGetSpec(t *testing.T) {
 		case "TERMINAL_BLOCK_HASH_ACTIVATION_EPOCH":
 			assert.Equal(t, "72", v)
 		case "TERMINAL_BLOCK_HASH":
-			assert.Equal(t, common.HexToHash("TerminalBlockHash"), common.HexToHash(v))
+			s, ok := v.(string)
```
Comment (james-prysm) [inline beacon-chain/rpc/eth/config/handlers_test.go:386]: there's a bunch of new ones for deneb are they already added?

Comment (rkapka) [reply beacon-chain/rpc/eth/config/handlers_test.go:386]: I will update this endpoint in another PR, some things are missing

File: proto/eth/v1/beacon_chain.proto [hunk 2]
```diff
@@ -20,7 +20,6 @@ import "google/protobuf/descriptor.proto";
 import "proto/eth/ext/options.proto";
 import "proto/eth/v1/attestation.proto";
 import "proto/eth/v1/beacon_block.proto";
```
Comment (james-prysm) [inline proto/eth/v1/beacon_chain.proto:22]: linting is saying this is not needed, ( a few of the imports) we can take care of it in a separate PR.

**Review by james-prysm — APPROVED** [review]
lgtm

**Pull request merged**